src/components/CheckoutButton/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { PayloCheckout } from './index.view';
```

Comment:
@KanRMUTL you can move checkoutButton (loadingButtonImage for proper name) to here. the image is related to this component.

Reply:
fixed

Reply:
@KanRMUTL cpuld you rename it ? checkoutButton should be loadingButtonImage  ? the image is loading image not checkout.

Reply:
I renamed checkoutButton to loadingButtonImage complete.